docs/UserGuide.md

```diff
@@ -402,6 +402,11 @@ Furthermore, certain edits can cause the application to behave in unexpected way
 --------------------------------------------------------------------------------------------------------------------
 <div style="page-break-after: always;"></div>
 
+## Glossary
+* **CLI**: Command-line interface where you interact with the system using your keyboard.
```

Comment:
I found this definition online [here ](https://www.shecodes.io/athena/138387-what-is-a-terminal-a-guide-to-the-command-line#:~:text=Answered%20in%201.45%20seconds,and%20perform%20various%20other%20tasks.)and thought this would be easier to understand for newbies? `allows you to interact with your computer through text-based commands. It provides a way to run programs, navigate through files and directories, manage processes, and perform various other tasks.`